stestr/commands/run.py

```diff
@@ -206,6 +207,21 @@ def run_tests():
             result = run_tests()
```

Comment:
oh, nope you're right. In my tests I was was checking that it fails eventually. Looking closely I see there are 2 failures now. So that's definitely what's happening. I'll respin this tomorrow to fix the bug